### DISCUSSION TITLE
Deployment ERROR ON ReactJS Redux Project

### DISCUSSION THREAD

**Discussion by cuongp1** [pdfme/pdfme#85] [Q&A]
Hi,  I am implement pdfme inside ReactJS, Redux project. Currently with versions:
1. Node 14.18.0
2. React 16.14.0
3. Redux 4.0.4
4. antd 3.26.20

On develop environment (npm-run-all -p watch-css start-js) everything working normally
But when i deployment for production (npm-run-all build-css pdfmake-font build-js) i got white screen error on page where using pdfme. 

Link image error: https://im.ge/i/FOkH6M

Please help, Thank you!

**Comment by hand-dot** [comment]
Hi @cuongp1 

I replied on this issue https://github.com/pdfme/pdfme/issues/83#issuecomment-1236583064
